slo-monitor/Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 PACKAGE = k8s.io/perf-tests/slo-monitor
-TAG = 0.14.0
+TAG = 0.14.1
```

Comment:
let's make 0.15 - it's a bit non-negligible change

Reply:
Done.